lib/components/src/tabs/tabs.tsx

```diff
@@ -114,7 +114,7 @@ export const panelProps = {
 
 const childrenToList = (children: any, selected: string) =>
   Children.toArray(children).map(
-    ({ props: { title, id, color, children: childrenOfChild } }, index) => {
+    ({ props: { title, id, color, children: childrenOfChild } }: React.ReactElement, index) => {
```

Comment:
Seems like before in the react typings this was only React.ReactElement, but in the new typings it can also be things like portals, and strings.

We'll likely have to deal with that at some point, but for now make the typings work.